statistics/handle/statscache.go

```diff
@@ -151,15 +192,20 @@ func (sc *statsCache) Update(tables []*statistics.Table, deletedIDs []int64, new
 	}
 }
 
-func (sc *statsCache) GetVersion() uint64 {
+// GetBytesLimit get the limits of memory.
+func (sc *simpleLRUStatsCache) GetBytesLimit() int64 {
```

Comment:
Lock()?

Reply:
It's no need, for now, tracker.bytelimit can not be reset after been initilized